app/config.py

```diff
@@ -39,16 +42,19 @@ class BaseConfig:
 
     HOST = env('HOST')
     PORT = env('PORT')
+    DEBUG = env('DEBUG', cast=bool, default=False)
     LOG_LEVEL = env('LOG_LEVEL')
     EXT_LOG_LEVEL = env('EXT_LOG_LEVEL')
 
     DOMAIN_URL_PROTOCOL = env('DOMAIN_URL_PROTOCOL', default='https://')
-    DOMAIN_URL_EN = env('DOMAIN_URL_EN')  # DOMAIN_URL_EN needs to be renamed to DOMAIN_URL
+    DOMAIN_URL = env('DOMAIN_URL')
```

Comment:
Still a few mentions of `DOMAIN_URL_EN` in the templates and other files. Worth doing that here or keeping it separate to the other card